releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java

```diff
@@ -125,6 +135,16 @@ public static Result buildResult(String localRepoPath, String authToken, String
                         remoteRepoPath, issueNo);
                     result.addError(error);
                 }
+
+                List<String> releaseLabels = Arrays.asList(getIssueLabelFrom(issue));
```

Comment:
I want to use it like:
```
                final String releaseLabels = getIssueLabelFrom(issue);
                if ( moreThanOneReleaseLabel(releaseLabels)) {
```

where moreThanOneReleaseLabel looks like:

```
    public static boolean moreThanOneReleaseLabel(List<String> releaseLabels) {
        return releaseLabels.size() > 1;
    }
```

but I can't do it because:
`java.lang.String cannot be converted to java.util.List<java.lang.String>`

Reply:
Changes Done. 

![dsvsdv](https://user-images.githubusercontent.com/71710042/180052485-cbe4e6b2-abd3-4158-a2b6-d90a04629b60.png)